pages/leadership_circle.js

```diff
@@ -53,10 +54,64 @@ export default () => (
         </div>,
       ]}
     />
+    <Content
+      title="Operation Code Honor Roll"
+      theme="gray"
+      columns={[
+        <Card className={styles.honorRollCard}>
+          <h6>Benefactor ($2500 or more)</h6>
+          <ul className={styles.honorRollList}>
+            <li>Conrad Hollomon</li>
```

Comment:
Note the **Other Information** aspect of the original ticket:
```
Non-developer members of Operation Code will need to update this data. Please ensure that the donors are placed into a sensible data structure as a separate file, and create a README tutorial next to that file to explain every single step needed for somebody to update the file. Assume the readers know nothing about GitHub, but are able to navigate the browser and will bookmark your README file.
```

So your'e missing two things

# 1

I recommend taking out all the names as hard-coded HTML and placing it into a `.json` file like so:

```json
{
  "benefactors": [
    "Conrad Hollomon",
    "Chris Todd",
    "Laura Wicket"
  ],
  "patrons": [
    "Liliana Monge',
    "Aaron Sahlstrom",
    "Nic and Gwyn Benders"
  ]
}
```

# 2

You need to create an in-depth README file for non-devs to understand how to update the JSON file.